samigo/samigo-qti/src/java/org/sakaiproject/tool/assessment/qti/helper/AuthoringHelper.java

```diff
@@ -800,7 +800,7 @@ public QuestionPoolFacade createImportedQuestionPool(Document document)
 
  	          while (!isUnique)
  	          {
- 	        	title = exHelper.renameDuplicate(title);
+ 	        	title = AssessmentService.renameDuplicate(title);
```

Comment:
assessmentService vs AssessmentService?

Reply:
On the previous bit there's already a variable for the service, but here we can directly call the static method. I can change any of them though...